### PR TITLE
Updated the readme file , changed the test description on the shipping and index files

### PR DESCRIPTION
**Pull request by CodingManoj**
Updated the readme file , changed the test description on the shipping and index files.

These were tested locally and working as intended, looking forward for the PR Review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,9 @@ Finally restart the service once to effect the changes.
 ```
 # systemctl restart nginx 
 ```
+
+ Check for the status of the service 
+
```
Comment (trainer-manuverma) [inline README.md:38]: Pls correct the spaces

File: localhost.conf [hunk 2]
```diff
@@ -8,7 +8,7 @@
     #error_page  404              /404.html;
 
     # redirect server error pages to the static page /50x.html
```
Comment (trainer-manuverma) [inline localhost.conf:10]: Remove the line 8

**Review by trainer-manuverma — CHANGES_REQUESTED** [review]
@CodingManoj  pls follow the commens and resend it to me

File: README.md (hunk 1 above)
Comment (CodingManoj) [reply README.md]: Spaced are corrected as per the standard.

File: localhost.conf (hunk 2 above)
Comment (CodingManoj) [reply localhost.conf:9]: Line 8 has been removed.

**Review by trainer-manuverma — APPROVED** [review]
Changes looks good to me

**Pull request merged**